src/sinks/nats.rs

```diff
@@ -242,39 +225,37 @@ mod tests {
 #[cfg(feature = "nats-integration-tests")]
 #[cfg(test)]
 mod integration_tests {
-    use std::{thread, time::Duration};
+    use std::{matches, thread, time::Duration};
```

Comment:
Is this use actually necessary? The rest of our code uses `matches!` without an explicit `use`.